client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -499,7 +503,38 @@ export const ApplicationsTable: React.FC = () => {
         ],
         getItemValue: (item) => normalizeRisk(item.risk) ?? "",
       },
+
+      {
+        categoryKey: "analysis",
+        title: t("terms.analysis"),
+        type: FilterType.multiselect,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.analysis").toLowerCase(),
+          }) + "...",
+
+        selectOptions: Object.values(applications)
+          .map((a) => {
```

Comment:
You will need to re-use the same mapper for sorting so let's create a named function for that.
The steps remain the same i.e. (pseudo-code)
```
const mapAnalysisStateToLabel = (value: TaskState) => {
   // for consistency let's use the same wa of converting as the rest of the code
   const presetKey: IconedStatusPreset = getTaskStatus(value);
  // as for now the presets are created inside the  IconedStatus
  // we need refactor this code and extract this functionality to a function i.e.buildPresets() 
  // this is best to do under a different PR 
   const presets = buildPresets(t);
   // IMHO fallback to "Unknown" makes most sense
   const label = presets[presetKey]?.label ?? t("terms.unknown");
   return label;
}
```
Links:
 [getTaskStatus(value)](https://github.com/konveyor/tackle2-ui/blob/c04bb35390b3bd06e425dc8cead7cd92df2dd478/client/src/app/pages/applications/components/application-analysis-status.tsx#L24)

Reply:
the functions mentioned here are pushed in #2125